src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/controller/ReferralController.kt

```diff
@@ -103,9 +104,13 @@ class ReferralController(
   @GetMapping("/sent-referrals")
   fun getSentReferrals(
     authentication: JwtAuthenticationToken,
+    @RequestParam(name = "completed", required = false) completed: Boolean?,
```

Comment:
i think it could be good to have the default values for these defined here? just reading it for the first time it wasn't clear what these would default to; i think you have to go right down to the spec to see that?